bool from_camera_to_base_link
---
float64 x_base_link_frame
float64 y_base_link_frame
float64 z_base_link_frame